src/NUnitFramework/framework/Internal/Execution/CompositeWorkItem.cs

```diff
@@ -99,9 +99,8 @@ protected override void PerformWork()
                         default:
                         case RunState.Runnable:
                         case RunState.Explicit:
-                            // Assume success, since the result will be inconclusive
-                            // if there is no setup method to run or if the
-                            // context initialization fails.
+                            // Assume success, since the result will otherwise
+                            // edault to inconclusive.
```

Comment:
I wouldn't normally go back for something as small as this, but I notice a lot of commented out code as well, which should be removed, so I'll do another push.